ister.py

```diff
@@ -1113,8 +1113,13 @@ def validate_user_template(users):
                                 .format(user))
 
         if key:
-            with open(user["key"], "r") as key_file:
-                user["key"] = key_file.read()
+            try:
+                with open(user["key"], "r") as key_file:
+                    user["key"] = key_file.read()
+            except FileNotFoundError:
```

Comment:
Having this be an exact error is good but what do you think of a general except case for problems with the file read?

Reply:
Well, the code did not handle the general OSError so I did not do it either. Kind of preserved old behavior. The old behavior was - any error reading the file - raise an exception.

If we cannot read the file for other reasons (permissions, or it is not a file, etc), the exception will be raised and caught in main(). The error message will be printed. For file I/O exceptions this typically includes file name. When debugging is enabled - we'll see also the stacktrace.

I do not think we should fall back to assuming this is an inline key for any exception. I think anything but FileNotFoundError is really a problem ans we should promptly fail rather than continue and do something users do not expect. If in the future we have users who have an issue with this, we can adjust the behavior. At this point it is good enough in my opinion.

Let me know if there is something specific in your mind.

Reply:
Ah right, sorry for the noise as I clearly temporarily forgot how python handles exceptions X(.